Firestore/Source/Public/FIRGeoPoint.h

```diff
@@ -39,7 +39,14 @@ NS_SWIFT_NAME(GeoPoint)
 - (instancetype)initWithLatitude:(double)latitude
                        longitude:(double)longitude NS_DESIGNATED_INITIALIZER;
 
+/**
+ * The point's latitude. Clamped to [-90, 90].
```

Comment:
Clamped isn't quite right because we reject arguments outside this range.

Android documents this as "The latitude of this GeoPoint."

If you really want to document the range, perhaps "The latitude of this point as a number between -90 and 90"? This duplicates the initializer comment but maybe is OK.

In any case, we should document latitude and longitude similarly: either the range should be included in both or not in both.

Reply:
Done. Opted for the wording `Must be a value between` for both fields.

Reply:
Test